pkg/rpc/server.go

```diff
@@ -328,6 +331,40 @@ func (s *Server) getAccountState(reqParams Params, unspents bool) (interface{},
 	return results, resultsErr
 }
 
+// invoke implements the `invoke` RPC call.
+func (s *Server) invoke(reqParams Params) (interface{}, error) {
+	scriptHashHex, ok := reqParams.ValueWithType(0, stringT)
+	if !ok {
+		return nil, errInvalidParams
+	}
+	scriptHash, err := scriptHashHex.GetUint160FromHex()
+	if err != nil {
+		return nil, err
+	}
+	sliceP, ok := reqParams.ValueWithType(1, arrayT)
+	if !ok {
+		return nil, errInvalidParams
+	}
+	slice, err := sliceP.GetArray()
+	if err != nil {
+		return nil, err
+	}
+	script, err := CreateInvocationScript(scriptHash, slice)
+	if err != nil {
+		return nil, err
+	}
+	vm, _ := s.chain.GetTestVM()
+	vm.LoadScript(script)
+	_ = vm.Run()
+	result := &wrappers.InvokeResult{
+		State:       vm.State(),
+		GasConsumed: "0.1",
```

Comment:
is that always 0.1? Can this be a constant with some reference to doc?

Reply:
Yes. No. #424.